datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java

```diff
@@ -996,6 +998,11 @@ private void validateShape(SdxClusterShape shape, String runtime, DetailedEnviro
                 validationBuilder.error("Provisioning a Medium Duty SDX shape is only valid for CM version greater than or equal to "
                         + MEDIUM_DUTY_REQUIRED_VERSION + " and not " + runtime);
             }
+        } else if (SdxClusterShape.SCALABLE.equals(shape)) {
```

Comment:
There a multiple places in the code that checks for medium duty. Shouldn't we review them to ensure the new scalable is covered in all those cases? Here is an example:

 public FlowIdentifier triggerRepairByCrn(String userCrn, String clusterCrn, SdxRepairRequest clusterRepairRequest) {
        SdxCluster cluster = sdxService.getByCrn(userCrn, clusterCrn);
        MDCBuilder.buildMdcContext(cluster);
        if (SdxClusterShape.MEDIUM_DUTY_HA.equals(cluster.getClusterShape()) && !entitlementService.haRepairEnabled(cluster.getAccountId())) {
            LOGGER.error("Cluster {} is Medium Duty and is not allowed to be repaired", cluster.getClusterName());
            throw new BadRequestException("Cannot repair Medium Duty cluster " + cluster.getClusterName() + " at this time");
        }
        return sdxReactorFlowManager.triggerSdxRepairFlow(cluster, clusterRepairRequest);
    }

Reply:
Done as part of https://github.com/hortonworks/cloudbreak/pull/14312/commits/9cd74bfccb87817599fbf276a43011f362082590 please let me know if it looks good.

Reply:
Done as part of https://github.com/hortonworks/cloudbreak/pull/14312/commits/9cd74bfccb87817599fbf276a43011f362082590 please let me know if it looks good.

Reply:
I left a comment. You want to separate this change from this PR?

Reply:
I do not mind keeping this here as it is simple enough. As for your comment:

```
How about validateShape in SdxService.java
```

The line of code your original comment is pointing to addresses `validateShape` already.